src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -40,8 +43,12 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
         contentSource: null,
         selectedRegions: [],
         canvasEnabled: true,
+        multiSelect: false,
     };
 
+    private clipBoard: ClipBoard<IRegion[]> = new ClipBoard<IRegion[]>();
+
+    private videoPlayer: React.RefObject<Player> = React.createRef<Player>();
```

Comment:
The video player does not exist directly in the canvas anymore and we shouldn't need this.